config.go

```diff
@@ -81,15 +81,16 @@ type config struct {
 }
 
 var conf = config{
-	Bind:            ":8080",
-	ReadTimeout:     10,
-	WriteTimeout:    10,
-	DownloadTimeout: 5,
-	Concurrency:     runtime.NumCPU() * 2,
-	TTL:             3600,
-	MaxSrcDimension: 4096,
-	Quality:         80,
-	GZipCompression: 5,
+	Bind:             ":8080",
+	ReadTimeout:      10,
+	WriteTimeout:     10,
+	DownloadTimeout:  5,
+	Concurrency:      runtime.NumCPU() * 2,
+	TTL:              3600,
+	MaxSrcDimension:  4096,
+	MaxSrcResolution: 65,
```

Comment:
65 megapixels is quite hight for default IMHO. Let's set it to 15.

Reply:
It used to be MaxSrcDimension* MaxSrcDimension, i.e. 16 megapixels.